static/stories/licenses.md

```diff
@@ -7,14 +7,69 @@ author: eScience Center
 thumbnail: "nlesc-dummy-1.png"
 ---
 
-:::Chapter{headline="Software licenses" image="ghlogo.png"}
-## Software licenses
+:::Chapter{headline="Introduction" image="ghlogo.png"}
+## Software licenses: introduction
 
-FOSS
+Copyright is a basic element of many things we produce, including software. Copyright, roughly speaking, means that we get to say how this work can be re-used by others, and what rights they have in copying, modifying, and distributing the work. When we do not explicitly formalize these rules in a license, the software cannot be (re-)used by others than the copyright holder. Including a license with software or code is therefore extremely important.
 
-Difference between permissive and viral licenses
+If someone codes in their own time for a project only they contribute to, then they are the copyright holder. When a project is authored by multiple people, all of these people have copyright, and therefore need to agree on the license.
 
-University policy
+When a project is done under employment, the copyright situation becomes more difficult. In most cases, copyright officially belongs to the employer, although the exact situation can differ depending on the employment contract. It is good practice therefore to check whether the employer has a policy for software licensing.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-:::
\ No newline at end of file
+In practice, it is often the case that (University) employees assume copyright of their own work, and in the absence of (open source) policy, decide on their own licenses. Being thorough and checking with the University's legal department may result in an overly cautious response that requires the use of a proprietary license, heavily limiting the options for software (re)use.
```

Comment:
This paragraph made me slightly uncomfortable although I really like the honesty. It almost sounds like you should not be thorough and consult your University's legal department because that will give you more trouble than you would like... It can be solved maybe by adding a sentence at the start of the next paragraph.

Reply:
I copied this from a module I wrote for Carpentries, which has a very different audience. In fact, I want to change the tone entirely and urge the target audience for this module to actively lobby for a university-wide policy. So, this is still filler-text and the actual content can be (largely ;)) ignored, I think.

If you're more comfortable if I remove the entire paragraph I'm happy to do so :)